src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -471,6 +471,10 @@ class FluxPipeline : public DiffusionPipeline {
 
         // Prepare timesteps
         std::vector<float> timesteps = m_scheduler->get_float_timesteps();
+        OPENVINO_ASSERT(!timesteps.empty(),
+                        "After adjusting the num_inference_steps by strength parameter: ", m_custom_generation_config.strength,
+                        " the number of pipeline steps is less then 1 and not appropriate for this pipeline. Please set a different strength value.");
```

Comment:
looks like it can be a generic check inside all schedulers when they return timesteps

Reply:
We have this strength adjusting for FlowMatchEulerDiscreteScheduler only

Reply:
each scheduler has such logic I suppose:
https://github.com/openvinotoolkit/openvino.genai/blob/b26c2592c845c7da8169a06792f0662687986482/src/cpp/src/image_generation/schedulers/euler_discrete.cpp#L197-L207